pkg/products/amqonline/reconciler.go

```diff
@@ -350,3 +359,43 @@ func (r *Reconciler) reconcileBlackboxTargets(ctx context.Context, installation
 
 	return integreatlyv1alpha1.PhaseCompleted, nil
 }
+
+func (r *Reconciler) reconcilePrometheusRule(ctx context.Context, installation *integreatlyv1alpha1.Installation, client k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
+	monitoringConfig := config.NewMonitoring(config.ProductConfig{})
+	rule := &monitoringv1.PrometheusRule{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "rhmi-amq-online-sli",
+			Namespace: r.Config.GetNamespace(),
+		},
+	}
+
+	rules := []monitoringv1.Rule{}
+	rules = append(rules, monitoringv1.Rule{
+		Alert: fmt.Sprintf("AMQ-SLI-1.1: AMQ Online console is not available in namespace %s", r.Config.GetNamespace()),
+		Annotations: map[string]string{
+			"sop_url": "https://github.com/RHCloudServices/integreatly-help/blob/master/sops/alerts_and_troubleshooting.md",
+			"message": "AMQ Online Console is not available",
+		},
+		Expr:   intstr.FromString(fmt.Sprintf("absent(kube_endpoint_address_available{endpoint=\"console\",namespace=\"%s\"}==1)", r.Config.GetNamespace())),
```

Comment:
Is this the query we want to use? or is there something better that we could replace it with? @david-martin @grdryn

Reply:
Leaving this as  `kube-endpoint-address-available`, as we discussed in person yesterday